src/handlers/int-test-support/helpers/mock-data/invoice/helpers.ts

```diff
@@ -28,3 +31,57 @@ export const createInvoiceInS3 = async (
     params.filename
   );
 };
+
+export const createInvoiceWithGivenData = async (
```

Comment:
I'll need to get back to you on a suggestion for how to move forward on this because it looks like a few different things hinge on this but I don't like the idea that there's this function that makes invoices and tests the standardisation flow and it's then thrown in all over the place

Reply:
if it works I suppose we should probably go ahead and get this merged so we have end to end tests but we should get something on the backlog to sort this

Reply:
Hi @Si3rr4wow  refactored this function, could you please have a look now?